src/pages/index.tsx

```diff
@@ -446,25 +413,25 @@ export default function Home(): JSX.Element {
                 </p>
               </li>
             </ul>
-            {/*<blockquote className={styles.quotation}>*/}
-            {/*  <div>*/}
-            {/*    <p>*/}
-            {/*      I installed Resoto. <strong>Super cool CLI experience</strong>*/}
-            {/*      , and I love the <strong>app configuration</strong>&mdash;much*/}
-            {/*      better than YAML hunting.*/}
-            {/*    </p>*/}
-            {/*    <footer>*/}
-            {/*      <strong>Justyn Roberts</strong>, Solutions Engineering at{' '}*/}
-            {/*      <a*/}
-            {/*        href="https://pagerduty.com"*/}
-            {/*        target="_blank"*/}
-            {/*        rel="noopener noreferrer"*/}
-            {/*      >*/}
-            {/*        PagerDuty*/}
-            {/*      </a>*/}
-            {/*    </footer>*/}
-            {/*  </div>*/}
-            {/*</blockquote>*/}
+            {/* <blockquote className={styles.quotation}>
+              <div>
+                <p>
+                  I installed Resoto. <strong>Super cool CLI experience</strong>
+                  , and I love the <strong>app configuration</strong>&mdash;much
+                  better than YAML hunting.
+                </p>
+                <footer>
+                  <strong>Justyn Roberts</strong>, Solutions Engineering at{' '}
+                  <a
+                    href="https://pagerduty.com"
+                    target="_blank"
+                    rel="noopener noreferrer"
+                  >
+                    PagerDuty
+                  </a>
+                </footer>
+              </div>
+            </blockquote> */}
```

Comment:
Do we have approval from the persons mentioned?

Reply:
I only fixed the comment syntax, they are still not visible on the site.